docs/static/configuration.asciidoc

```diff
@@ -510,18 +510,45 @@ doesn't exist versus a field that's simply false. The expression `if [foo]` retu
 
 For more complex examples, see <<using-conditionals, Using Conditionals>>.
 
-[float]
+[discrete]
+[[date-math]]
+===== Date math in conditionals
+
+Date math in conditionals is not currently supported, but a workaround is available. 
```

Comment:
If we put this content below `@metadata`, maybe we can refer to it here, stating something like "a workaround using the `@metadata` field is available"

Unless that is a bit long winded and unnecessary detail